### PR TITLE
Add instructions for poetry shell and pyenv

### PR DESCRIPTION
**Pull request by rodrigovieira-vinta**
While I was setting up the project locally, I had an issue with the version of Python required. This PR adds some instructions on how to use `pyenv` to handle this issue.

Additionally, it adds instructions on how to use `poetry shell` to instantiate the virtual env specified on `pyproject.toml`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,22 +12,37 @@ Please follow this guide to learn more about how to develop and test the project
 git clone git@github.com:vintasoftware/django-ai-assistant.git
 ```
 
-### Set up a virtualenv, optionally set up nvm, and activate your environment(s)
+### Install development tools
 
-You can use [pyenv](https://github.com/pyenv/pyenv), [pipenv](https://github.com/pypa/pipenv/blob/main/docs/installation.md), vanilla venvs or the tool of your choice.
+This project uses [Poetry](https://python-poetry.org/docs/) for dependency and virtual environment management.
+
+If you need to install the version of Python recommented for the project, you can use [Pyenv](https://github.com/pyenv/pyenv).
```
Comment (fjsj) [inline CONTRIBUTING.md:19]: Typo: recommented -> recommended

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall LGTM, but there's a small typo

**Review by fjsj — APPROVED** [review]

**Comment by filipeximenes** [comment]
Ugh, I didn't see this PR and opened this [other one](https://github.com/vintasoftware/django-ai-assistant/pull/158) which adds the instruction to activate `poetry shell`, sorry.

**Comment by fjsj** [comment]
Can you make sure we also drop the mention to pipenv? @filipeximenes

**Comment by filipeximenes** [comment]
@fjsj will do. @rodbv I can't commit on your repository so I'll pull from your branch and open a new PR with these changes ok?

**Comment by rodrigovieira-vinta** [comment]
> @fjsj will do. @rodbv I can't commit on your repository so I'll pull from your branch and open a new PR with these changes ok?

Sure!

**Pull request merged**